projects/packages/search/src/dashboard/components/record-meter/bar-chart.jsx

```diff
@@ -0,0 +1,158 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import Chart from 'chart.js/auto';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+export class BarChart extends React.Component {
+	constructor( props ) {
+		super( props );
+		this.canvasRef = React.createRef();
+	}
+
+	componentDidMount() {
+		const CHART_OPTIONS = {
```

Comment:
Could we move this declaration outside of the class definition (e.g. right below the `__` import at line 11?)?

Reply:
Fixed in b146e71 👍